src/modules/common/components/Selector.styles.js

```diff
@@ -30,7 +24,19 @@ export const Element = styled(Link)`
   }
 `
 
-export const ActiveElement = Element.withComponent('span').extend`
+export const ActiveElement = Element.extend`
+  :hover {
+    font-weight: 700;
+    border-radius: 0;
+  }
+  color: ${props => props.theme.colors.textColor};
+`
+
+export const InactiveElement = Element.withComponent('span').extend`
+  color: ${props => props.theme.colors.textSecondaryColor};
```

Comment:
Maybe we can even go with `textDecorativeColor` but I'm not sure... For me `textSecondaryColor` is still too dark and not easy to distinguish with `textColor`... Maybe we need a `textDisabledColor` :D

Reply:
I agree, could you create a new task?